11.0.Dockerfile

```diff
@@ -29,6 +29,11 @@ ENV DB_FILTER=.* \
     WDB_WEB_PORT=1984 \
     WDB_WEB_SERVER=localhost
 
+# Enable configuring geoip with scaffolding environment (by setting GEOIP_ACCOUNT_ID and GEOIP_LICENSE_KEY)
+ENV GEOIP_ACCOUNT_ID="" \
+    GEOIP_LICENSE_KEY=""
+ARG GEOIP_UPDATER_VERSION=4.1.5
```

Comment:
Please move ARGs to the top (see where the others are).

Then please join that new ENV with the ENV clause above.

Try to sort them alphabetically.

This is to make good usage of cache and reduce layers.

Reply:
i moved it to the top